app/models/manageiq/providers/hawkular/datawarehouse_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,105 @@
+class ManageIQ::Providers::Hawkular::DatawarehouseManager::EventCatcher::Runner <
```

Comment:
/cc @jshaughn mainly designed the events catcher in the Middleware provider for ManageIQ.
Jay, do you think that it is doable to reuse the mixins they are referenced in the discussion ?
Could you take a look on them ?